pod-node/README.md

```diff
@@ -0,0 +1,21 @@
+# Pod-Node
+
+It's a native node module, wraps the pod-library.
+
+# Prerequisites
+
+After everything is ready with SGX _(check "pod-server" folder README)_, be sure you have;
+
+- node (from https://nodejs.org, or preffered https://github.com/nvm-sh/nvm)
+- node-gyp (npm install -g node-gyp)
+
+- SPID from Intel
+- Put your SPID in `src/module.js`
```

Comment:
Hmm, this might be confusing. I think it’d be better to say something like “the SPID can be obtained from the service provider, i.e., the entity that runs the web service `pod-server`”.

Reply:
Yes, I will improve the Readme, haven't add the url too.